features/geometrisch-zoeken.feature

```diff
@@ -0,0 +1,327 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik met een geometrie (punt, bounding box) kunnen zoeken naar resources, 
+                 zodat ik gericht vervolgstappen kan definiëren m.b.t. de gevonden resources.
+
+    De resources waarvan de geometrie tenminste één punt gemeenschappelijk hebben met de opgegeven geometrie worden geretourneerd.
+
+    Bij het bepalen of een resource geometrie binnen de opgegeven contour ligt (bbox of polygoon) of 
+    dat een opgegeven geometrie (punt) binnen de resource geometrie ligt, wordt een intersects spatial operator gebruikt.
+
+    Intersects:
+    Om te bepalen of een geometrie A binnen een geometrie B ligt, deze raakt, kruist of (deels) overlapt, wordt gebruik gemaakt van de spatial operator INTERSECTS:
+
+      intersects(geometrie A, geometrie B)
+
+    Intersects betekent dat geometrie A en geometrie B tenminste één gemeenschappelijk punt hebben.
+    Anders gezegd geometrie A en geometrie B mogen niet volledig los van elkaar staan.
+    Het resultaat van intersects(geometrie A, geometrie B) levert hetzelfde op als intersects(geometrie B, geometrie A).
+    Het maakt niet uit of geometrie A of geometrie B de geometrie is waarmee gezocht wordt en
+    het maakt niet uit of geometrie A of geometrie B de resource geometrie is.
+    Als intersects true oplevert, dan wordt de resource waar de geometrie onderdeel van is, geretourneerd.
+
+    Referenties:
+    - Haal Centraal Common - foutafhandeling.feature
+    
+    Het geometrisch zoeken met een bounding box geometrie kan met de parameter: bbox.
+    
+    Het geometrisch zoeken met een polygoon geometrie kan met de parameter: polygoon.
+
+    Het geometrisch zoeken met een punt geometrie kan met de parameter: locatie.
+    Het is echter ook mogelijk om een bounding box op te geven waarbij de coördinaten van de linker onderhoek en de rechter bovenhoek exact gelijk zijn.
+
+    In de hierna volgende voorbeelden is een vereenvoudigde representatie gebruikt voor coördinaten.
+    
+    ![Zoeken met locatie voorbeelden](../images/locatie-voorbeelden.png)    
+    ![Zoeken met bounding box voorbeelden](../images/bbox-voorbeelden.png)    
+    ![Zoeken met polygoon voorbeelden](../images/locatie-voorbeelden.png)      
+
+  Achtergrond:
+
+    Gegeven de volgende verzameling van objecten
+
+    | identificatie | geometrie                          |
+    |---------------|------------------------------------|
+    | 0001          | (0,0),   (10,0),  (10,10), (0,10)  |
+    | 0002          | (12,0),  (22,0),  (22,10), (12,10) |
+    | 0003          | (12,12), (22,12), (22,22), (12,22) |
+    | 0004          | (0,8),   (10,8),  (10,18), (0,18)  |
```

Comment:
de gegeven objecten hebben allemaal een box/polygon. Zoeken van verblijfsobjecten (geometrie is een punt) wordt niet geïllustreerd. Kan je dat ook toevoegen?